sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Custom/Models/PostgreSqlFlexibleServerHyperscaleNodeEditionCapability.cs

```diff
@@ -1,16 +1,18 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 // <auto-generated/>
```

Comment:
if we moved something to customized folder, we should also remove this.

Reply:
Done